### PR TITLE
[zh] Sync secret.md

### PR DESCRIPTION
**Pull request by Sea-n**
Re-sync some en content and add link anchor.

### CODE REVIEW

File: content/zh/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -1379,14 +1383,14 @@ Kubernetes 提供若干种内置的类型，用于一些常见的使用场景。
 
 <!--
 You can define and use your own Secret type by assigning a non-empty string as the
-`type` value for a Secret object. An empty string is treated as an `Opaque` type.
+`type` value for a Secret object (an empty string is treated as an `Opaque` type).
 -->
 通过为 Secret 对象的 `type` 字段设置一个非空的字符串值，你也可以定义并使用自己
-Secret 类型。如果 `type` 值为空字符串，则被视为 `Opaque` 类型。
+Secret 类型（如果 `type` 值为空字符串，则被视为 `Opaque` 类型。）
```
Comment (ydFu) [inline content/zh/docs/concepts/configuration/secret.md:1389]:
```suggestion
Secret 类型（如果 `type` 值为空字符串，则被视为 `Opaque` 类型）。
```

Comment (Sea-n) [reply content/zh/docs/concepts/configuration/secret.md]: Fixed, thanks.

**Pull request merged**